MuvrKit/MKClassifier.swift

```diff
@@ -28,6 +28,7 @@ public struct MKClassifier {
     private let neuralNet: MKForwardPropagator
     private let windowSize = 400
     private let windowStepSize = 10
+    private let scalingFactor = Float(4.0)
```

Comment:
The data from the accelerometer is in G as far as I know and in the range of -16 to 16. The values we get are normally between -4 and 4 and hence the scaling. Since the scaling factor is quite low, there is not a huge difference in training, but nevertheless the scaling between -1 and 1 is just the standard way of doing things. 

We also need to make sure we scale the data in the same way during training and classification.